src/pycountry/db.py

```diff
@@ -53,18 +57,21 @@ def __init__(self, filename):
         self._is_loaded = False
         self._load_lock = threading.Lock()
 
+        # create data class
+        self.data_class = type(self.data_class_name, (self.data_class_base,), {})
+
+    def _clear(self):
```

Comment:
I pulled out `_clear` as a separate method, mainly because it was useful for testing to have a capability to "reset" the databases. It could even be useful to make a `reset` part of the public API in the future, which allows you to revert any changes you've made with the new add and remove functionality.